firewall/dns.go

```diff
@@ -205,30 +205,49 @@ func mayBlockCNAMEs(conn *network.Connection) bool {
 	return false
 }
 
+// updateIPsAndCNAMEs saves all the IP->Name mappings to the cache database and
+// updates the CNAMEs in the Connection's Entity.
 func updateIPsAndCNAMEs(q *resolver.Query, rrCache *resolver.RRCache, conn *network.Connection) {
-	// save IP addresses to IPInfo
+	// FIXME: ignore localhost
+
+	// Get IPInfo scope.
+	var scope string
+	proc := conn.Process()
+	if proc != nil {
+		scope = proc.LocalProfileKey
+	}
+
```

Comment:
I'm confused about our scopes. I thought that can either be a domain or one of those two-letter constants? Now it can be a profile key as well?

I guess this scope is something else so maybe rename the variable?